lang/compiler/compile.go

```diff
@@ -116,7 +122,7 @@ func compileToObject(assemblerFilename string, objectFilename string) error {
 
 func compileToExecutable(objectFileName string, executableFileName string) error {
 	clang := findCommand("clang-19", "clang")
-	linkCmd := exec.Command(clang, objectFileName, "-o", executableFileName)
+	linkCmd := exec.Command(clang, objectFileName, "-o", executableFileName, "-no-pie")
```

Comment:
Are we sure we want to do this ?

```suggestion
	linkCmd := exec.Command(clang, objectFileName, "-o", executableFileName)
```